### PR TITLE
Add svg file to the repo and reference it

### PR DESCRIPTION
**Pull request by j-rivero**
Target https://github.com/RobotecAI/o3de-ros2-gem/pull/358

### CODE REVIEW

File: docs/design/manipulation/roadmap.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This roadmap outlines the implementation steps for the light version of the
 manipulation diagram (simplified by danielemorra98):
 
-![manipulation_light](https://user-images.githubusercontent.com/2098802/207460909-e3aac9a6-22be-4ce9-a6e8-27b970a16365.svg)
+![manipulation_light](manipulation_light.svg)
```
Comment (mabelzhang) [inline docs/design/manipulation/roadmap.md]:
```suggestion
![manipulation_light](manipulation_minimal.svg)
```

**Comment by mabelzhang** [comment]
Oops I didn't see this before addressing the comment in the PR! We can merge this too.

**Review by mabelzhang — APPROVED** [review]
Link typo

**Comment by j-rivero** [comment]
Should be fixed thanks, could you please merge the PR?

**Pull request merged**